### ISSUE TITLE
ESPnet2 Librispeech Conformer plot WER

### ISSUE THREAD

**Issue by qdenisq** [espnet/espnet#2732]
Hi, I have a question about the pretrained Conformer for LibriSpeech [link to the model](https://zenodo.org/record/4276519#.X8WpJ80zaUk). I wondered why the reported WER on the dev other is ~ 5.5%, whereas the WER for the validation set according to the plot set reaches ~ 45%. 

![wer](https://user-images.githubusercontent.com/14143468/100690310-5234a100-33da-11eb-97ba-8f142897407b.png)

Update: I checked the WER calculation during training (in `espnet2/asr/espnet_model.py`)  and it seems to be based on greedy decoding. 

```python
ys_hat = decoder_out.argmax(dim=-1)
cer_att, wer_att = self.error_calculator(ys_hat.cpu(), ys_pad.cpu())
```

The final WER in the table is calculated using beam search with lm if any. @sw005320, Could you please verify my assumption that the plotted WER is so much worse due to the decoding type?

### WER

|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
|---|---|---|---|---|---|---|---|---|
|decode_asr_lm_lm_train_lm_adam_bpe_valid.loss.ave_asr_model_valid.acc.ave/dev_clean|2703|54402|98.0|1.8|0.2|0.3|2.3|27.6|
|decode_asr_lm_lm_train_lm_adam_bpe_valid.loss.ave_asr_model_valid.acc.ave/dev_other|2864|50948|95.0|4.3|0.6|0.5|**5.5**|45.1|
|decode_asr_lm_lm_train_lm_adam_bpe_valid.loss.ave_asr_model_valid.acc.ave/test_clean|2620|52576|97.9|1.9|0.2|0.3|2.4|28.2|
|decode_asr_lm_lm_train_lm_adam_bpe_valid.loss.ave_asr_model_valid.acc.ave/test_other|2939|52343|94.9|4.5|0.7|0.6|5.8|48.6|

**Comment by kan-bayashi** [comment]
Since we use BPE for librispeech, WER during the training represents sentence error rate.

**Comment by qdenisq** [comment]
right, so if we wanted to get the actual WER and CER we would need to decode both `ys_hat` and `ys_pad` bpe sequences back into text and then pass it to `error_calculator`? btw many thanks for your amazing project
